externals/kyuubi-hive-sql-engine/src/main/scala/org/apache/kyuubi/engine/hive/session/HiveSessionImpl.scala

```diff
@@ -33,7 +33,6 @@ class HiveSessionImpl(
     protocol: TProtocolVersion,
     user: String,
     password: String,
-    serverIpAddress: String,
```

Comment:
this change may breaks https://github.com/apache/incubator-kyuubi/pull/1530/files#diff-759a09fe90c07b0ef78f6f4dc28bac9e3d0785d4b922cbe33422524e8a2520d4R51

Reply:
cc @turboFei @wForget

Reply:
[SparkSessionImpl.scala#L51](https://github.com/turboFei/incubator-kyuubi/blob/97f9b00417f211cc547998821ba421b15edb6c76/externals/kyuubi-spark-sql-engine/src/main/scala/org/apache/kyuubi/engine/spark/session/SparkSessionImpl.scala#L51) becomes redundant, thanks, I will remove it.

```
  def serverIpAddress(): String = serverIpAddress
```